packages/utils/ast-utils.js

```diff
@@ -382,26 +389,37 @@ function getRequire(j, constName, packagePath) {
  * @param {Node} p - AST node
  * @param {String} key - key of a key/val object
  * @param {Any} value - Any type of object
+ * @param {Boolean} update - Flag to check if node needs to update
  * @returns {Node} - the created ast
  */
 
-function addProperty(j, p, key, value) {
-	let valForNode;
+function addProperty(j, p, key, value, update = false) {
```

Comment:
This should be action, not `update`. Adjust to general cases, not the ones that fits your needs now. It might haunt you l8ter...